engine/common/synchronization/engine.go

```diff
@@ -610,7 +603,6 @@ func (e *Engine) onBlockResponse(originID flow.Identifier, res *messages.BlockRe
 		}
 		e.comp.SubmitLocal(synced)
 	}
-	return nil
```

Comment:
What's the logic for not inspecting errors and at least logging them here?

Reply:
same here. there is no error ever returned